contracts/contracts/vault/VaultCore.sol

```diff
@@ -433,7 +439,7 @@ contract VaultCore is VaultInitializer {
         returns (uint256 value)
     {
         uint256 assetCount = allAssets.length;
-        for (uint256 y = 0; y < assetCount; ++y) {
```

Comment:
Why are these explicit zero sets removed? They were added elsewhere in the PR? (I'm in camp explicit 0 sets)